shadow-dom/focus/focus-slot-box-generated-tabindex-0.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>HTML Test: focus - slot with tabindex=0 that generates a box should be focusable</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation">
```

Comment:
You can try adding 

```suggestion
<meta name=timeout content=long>
<link rel="help" href="https://html.spec.whatwg.org/multipage/interaction.html#sequential-focus-navigation">
```

which gives the test extra time to run, although I'd be surprised if it was taking that long to run.

Reply:
Ok, it actually seems like all of the tests like this are timing out for me locally, but on wpt.fyi they are running correctly and as expected for this test: Chrome fails while Safari and Firefox pass. I think I _might_ have the wrong help/spec link in the test (it was copied from another test), but I think this is ready for review now. Thanks for the help though!